### PR TITLE
Plurmy recipe

### PR DESCRIPTION
**Pull request by Kelvinrr**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Kelvinrr** [comment]
This should be good for review @conda-forge/help-python

**Comment by Kelvinrr** [comment]
@dopplershift thanks for the prompt review, comments addressed. I honestly don't know how to address this windows failure, if it has an easy fix I can take care of it (doesn't seem like a missing package error) otherwise we can ignore it.

File: recipes/plurmy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url:  https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c77f2b6ed9f6779ebf33ece1c7b8f43ec3057abfb5d621b0471e3afd6efd3790
+
+build:
+  skip: true  # [win]
```
[the remark this answers is not in the document]
Comment (dopplershift) [reply recipes/plurmy/meta.yaml]: You missed adding the `entry_points` section under build I suggested here.

**Comment by Kelvinrr** [comment]
Thanks, addressed it, idk how I missed that.

**Comment by Kelvinrr** [comment]
Just gonna ping for visibility since comments have been addressed: @dopplershift

**Review by dopplershift — APPROVED** [review]

**Pull request merged**